src/screens/auth/login/Login.tsx

```diff
@@ -327,117 +352,121 @@ export default class Login extends BaseScreen<Props, State> {
     const { tenantLogo, eula, loading, initialLoading, hidePassword } = this.state;
     // Render
     return initialLoading ? (
-      <Spinner style={formStyle.spinner} color='grey' />
+      <Spinner style={formStyle.spinner} color='grey'/>
     ) : (
-        <View style={style.container}>
```

Comment:
Be careful with reformatting, it generates a lot of changes in the diff tool with no update.
This one could have been kept in one line so not needed.
You can revert.